hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/BucketObjectDBInfo.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.recon.api.types;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.hadoop.hdds.client.DefaultReplicationConfig;
+import org.apache.hadoop.hdds.protocol.StorageType;
+import org.apache.hadoop.ozone.om.helpers.BucketEncryptionKeyInfo;
+import org.apache.hadoop.ozone.om.helpers.BucketLayout;
+import org.apache.hadoop.ozone.om.helpers.OmBucketInfo;
+
+/**
+ * Encapsulates the low level bucket info.
+ */
+public class BucketObjectDBInfo extends ObjectDBInfo {
+  @JsonProperty("volumeName")
+  private String volumeName;
+
+  @JsonProperty("storageType")
+  private StorageType storageType;
+
+  @JsonProperty("versioning")
+  private boolean isVersioningEnabled;
+
+  @JsonProperty("usedBytes")
+  private String usedBytes;
+
+  @JsonProperty("encryptionInfo")
+  private BucketEncryptionKeyInfo bekInfo;
+
+  @JsonProperty("replicationConfigInfo")
+  private DefaultReplicationConfig defaultReplicationConfig;
+
+  @JsonProperty("sourceVolume")
+  private String sourceVolume;
+
+  @JsonProperty("sourceBucket")
+  private String sourceBucket;
+
+  @JsonProperty("bucketLayout")
+  private BucketLayout bucketLayout;
+
+  @JsonProperty("owner")
+  private String owner;
+
+  public static BucketObjectDBInfo.Builder newBuilder() {
```

Comment:
Because `new BucketObjectDBInfo.Builder();` can be used instead of `BucketObjectDBInfo.newBuilder() `. It is not providing any benefit. 

If it is general pattern followed in the code base, please go ahead with it.

Reply:
@hemantk-12  - As discussed, I have implemented without builder and with plain constructor. Pls re-review.